test/ctia/test_helpers/core.clj

```diff
@@ -197,12 +195,12 @@
 (defmacro deftest-for-each-fixture [test-name fixture-map & body]
   `(do
      ~@(for [[name-key fixture-fn] fixture-map]
-         `(clojure.test/deftest ~(with-meta (symbol (str test-name "-" (name name-key)))
-                                   {(keyword name-key) true})
+         `(test/deftest ~(with-meta (symbol (str test-name "-" (name name-key)))
```

Comment:
Not sure if aliases in macros is an improvement - if I removed the alias from this file's `ns` form, the macro would break without a compile-time error

Whereas fqn is less likely to be broken

Reply:
Thanks for the thoughts. I'm aware of the tradeoffs, and IMO it's more robust overall to use aliases.